[2023-11-26T19:34:37.879+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.merge_data manual__2023-11-26T19:23:38.629095+00:00 [queued]>
[2023-11-26T19:34:37.883+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.merge_data manual__2023-11-26T19:23:38.629095+00:00 [queued]>
[2023-11-26T19:34:37.884+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2023-11-26T19:34:37.892+0000] {taskinstance.py:1380} INFO - Executing <Task(MergeDataOperator): merge_data> on 2023-11-26 19:23:38.629095+00:00
[2023-11-26T19:34:37.899+0000] {standard_task_runner.py:57} INFO - Started process 251 to run task
[2023-11-26T19:34:37.905+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'merge_data', 'manual__2023-11-26T19:23:38.629095+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp4sjk00vk']
[2023-11-26T19:34:37.908+0000] {standard_task_runner.py:85} INFO - Job 80: Subtask merge_data
[2023-11-26T19:34:37.940+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_dag.merge_data manual__2023-11-26T19:23:38.629095+00:00 [running]> on host d3fc1b5682ee
[2023-11-26T19:34:37.990+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='merge_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-26T19:23:38.629095+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-26T19:23:38.629095+00:00'
[2023-11-26T19:34:38.193+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=merge_data, execution_date=20231126T192338, start_date=20231126T193437, end_date=20231126T193438
[2023-11-26T19:34:38.236+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-26T19:34:38.261+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
